recipes/qt_binder/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2" %}
+
+package:
+  name: qt_binder
+  version: {{ version }}
+
+source:
+  fn: qt_binder-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/q/qt_binder/qt_binder-{{ version }}.tar.gz
+  sha256: 392a105a6a1734e1f67e17b88ee72225be594dfc0000dc3d53e254a1b8a532e5
+
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - libpng >=1.6.22  # current qt 4.8.7 3 from defaults requires libpng 1.6.22
```

Comment:
@grlee77 maybe you should use `libpng 1.6.22` to pin it exactly to avoid possible issues with `libpng 1.6.23` and up. I am not sure if that is needed, but `libpng` has history of incompatibility between its minor versions that is quite annoying.